### PR TITLE
Settings UX

### PR DESCRIPTION
**Pull request by marcua**
Doing the easy parts of @meredithblumenstock's feedback

### CODE REVIEW

**Comment by marcua** [comment]
Pending a bugfix we reported (https://github.com/stefanfoulis/django-phonenumber-field/issues/125), this should be good to go:

image

image

image

File: orchestra/static/accounts/scss/_registration.scss [hunk 1]
```diff
@@ -4,6 +4,9 @@
 /* Login/account-related pages */
 .login-wrap {
   font-size: 1.3rem;
+  .account-tabs selected {
```
Comment (joshblum) [inline orchestra/static/accounts/scss/_registration.scss]: rm? or is this used to clear the previous css?

**Comment by joshblum** [comment]
one question but otherwise lgtm

File: orchestra/static/accounts/scss/_registration.scss (hunk 1 above)
Comment (marcua) [inline orchestra/static/accounts/scss/_registration.scss]: good catch fixed!

**Comment by meredithblumenstock** [comment]
1)  Add more padding between the checkboxes and the checkbox labels
2) Maybe add more padding between the country code dropdown and the phone number text field (low priority)
3) Can you change the card color to match the white shade of the tab?

**Pull request merged**